e2e/docker/dex-contracts/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM node:lts
 
 WORKDIR /app/dex-contracts
 
-RUN git clone --depth 1 --branch v0.4.3 https://github.com/gnosis/dex-contracts . &&\
+RUN git clone --depth 1 --branch master https://github.com/gnosis/dex-contracts . &&\
```

Comment:
I'm slightly concerned that changes to dex-contracts might break the build here if we don't use tags.